specviz/plugins/line_labels/line_labels_plotter.py

```diff
@@ -1,89 +1,135 @@
 import numpy as np
 
 from qtpy.QtCore import QEvent, Qt, QThread, Signal, QMutex, QTime
+from qtpy.QtWidgets import QMessageBox
 
-from ..core.annotation import LineIDMarker, LineIDMarkerProxy
-from ..core.linelist import LineList, \
-    REDSHIFTED_WAVELENGTH_COLUMN, MARKER_COLUMN, ID_COLUMN, COLOR_COLUMN, HEIGHT_COLUMN
+from astropy import units as u
 
-__all__ = ['LineLabelsPlotter', 'ZoomMarkersThread', 'ZoomEventBuffer']
+from .annotation import LineIDMarker, LineIDMarkerProxy
+from .linelist import LineList, WAVELENGTH_COLUMN, \
+    REDSHIFTED_WAVELENGTH_COLUMN, \
+    MARKER_COLUMN, ID_COLUMN, COLOR_COLUMN, HEIGHT_COLUMN
 
 
 class LineLabelsPlotter(object):
```

Comment:
This file is also missing an ``__all__``

Reply:
Done.